core/psh/sysexec.c

```diff
@@ -25,15 +27,90 @@ void psh_sysexecinfo(void)
 }
 
 
+static int psh_sysexec_argmatch(const char *cmd, size_t cmdlen, char const **argv, int argc)
```

Comment:
nitpicking: why not-obvious args order: `argv, argc` instead of `argc, argv`? Also: please write the `const` berfore the `char` (so: `const char **argv`)

Reply:
Order of arguments changed. Thanks